stdlib/universe/universe.flux

```diff
@@ -237,6 +237,97 @@ builtin string : (v: A) => string
 builtin time : (v: A) => time
 builtin uint : (v: A) => uint
 
+// display returns the Flux literal representation of any value as a string.
+//
+// Basic types are converted directly to a string.
+// Bytes types are represented as a string of lowercase hexadecimal characters prefixed with `0x`.
+// Composite types (arrays, dictionaries, and records) are represented in a syntax similar
+// to their equivalent Flux literal representation.
+//
+// Note the following about the resulting string representation:
+// - It cannot always be parsed back into the original value.
+// - It may span multiple lines.
+// - It may change between Flux versions.
+//
+//`display()` differs from `string()` in that `display()` recursively converts values inside
+// composite types to strings. `string()` does not operate on composite types.
+//
+//  ## Parameters
+//  - v: Value to convert for display.
+//
+//  ## Examples
+//
+//  ### Display a value as part of a table
+//
+//  Using `array.from()` and `display() you can quickly observe any value.
```

Comment:
```suggestion
//  Use `array.from()` and `display()` to quickly observe any value.
```

Reply:
Thanks I'll figure out how to do this correctly someday.